drivers/dummy/display_server_dummy.h

```diff
@@ -0,0 +1,126 @@
+
+	void window_set_rect_changed_callback(const Callable &p_callable, WindowID p_window = MAIN_WINDOW_ID) override {}
+
+	void window_set_window_event_callback(const Callable &p_callable, WindowID p_window = MAIN_WINDOW_ID) override {}
+	void window_set_input_event_callback(const Callable &p_callable, WindowID p_window = MAIN_WINDOW_ID) override {}
+	void window_set_input_text_callback(const Callable &p_callable, WindowID p_window = MAIN_WINDOW_ID) override {}
+	void window_set_drop_files_callback(const Callable &p_callable, WindowID p_window = MAIN_WINDOW_ID) override {}
+
+	void window_set_title(const String &p_title, WindowID p_window = MAIN_WINDOW_ID) override {}
+
+	void window_set_mouse_passthrough(const Vector<Vector2> &p_region, WindowID p_window = MAIN_WINDOW_ID) override {}
+
+	int window_get_current_screen(WindowID p_window = MAIN_WINDOW_ID) const override { return -1; }
+	void window_set_current_screen(int p_screen, WindowID p_window = MAIN_WINDOW_ID) override {}
+
+	Point2i window_get_position(WindowID p_window = MAIN_WINDOW_ID) const override { return Point2i(); }
+	void window_set_position(const Point2i &p_position, WindowID p_window = MAIN_WINDOW_ID) override {}
+
+	void window_set_transient(WindowID p_window, WindowID p_parent) override {}
+
+	void window_set_max_size(const Size2i p_size, WindowID p_window = MAIN_WINDOW_ID) override {}
+	Size2i window_get_max_size(WindowID p_window = MAIN_WINDOW_ID) const override { return Size2i(); }
+
+	void window_set_min_size(const Size2i p_size, WindowID p_window = MAIN_WINDOW_ID) override {}
+	Size2i window_get_min_size(WindowID p_window = MAIN_WINDOW_ID) const override { return Size2i(); };
+
+	void window_set_size(const Size2i p_size, WindowID p_window = MAIN_WINDOW_ID) override {}
+	Size2i window_get_size(WindowID p_window = MAIN_WINDOW_ID) const override { return Size2i(); }
+	Size2i window_get_real_size(WindowID p_window = MAIN_WINDOW_ID) const override { return Size2i(); }
+
+	void window_set_mode(WindowMode p_mode, WindowID p_window = MAIN_WINDOW_ID) override {}
+	WindowMode window_get_mode(WindowID p_window = MAIN_WINDOW_ID) const override { return WINDOW_MODE_MINIMIZED; }
+
+	bool window_is_maximize_allowed(WindowID p_window = MAIN_WINDOW_ID) const override { return false; }
+
+	void window_set_flag(WindowFlags p_flag, bool p_enabled, WindowID p_window = MAIN_WINDOW_ID) override {}
+	virtual bool window_get_flag(WindowFlags p_flag, WindowID p_window = MAIN_WINDOW_ID) const override { return false; }
+
+	void window_request_attention(WindowID p_window = MAIN_WINDOW_ID) override {}
+	void window_move_to_foreground(WindowID p_window = MAIN_WINDOW_ID) override {}
+
+	bool window_can_draw(WindowID p_window = MAIN_WINDOW_ID) const override { return false; }
+
+	bool can_any_window_draw() const override { return false; }
+
+	void process_events() override {}
+
+	static void register_headless_driver() {
+		register_create_function("headless", create_func, get_rendering_drivers_func);
```

Comment:
Since we name it `headless` internally, maybe the implementation should be `DisplayServerHeadless`?

Reply:
Yeah, I think that's a good idea.